src/users/users.controller.ts

```diff
@@ -283,6 +285,42 @@ export class UsersController {
     }
   }
 
+  @Get('/one/:email/isadmin')
```

Comment:
Avoid using verbs as part of the path while designing RESTFul API 

```suggestion
  @Get('/one/:email/permissions')
```